src/private/CABasicAnimation+MotionAnimator.m

```diff
@@ -44,6 +44,16 @@ static BOOL IsCGSizeType(id someValue) {
   return NO;
 }
 
+static BOOL IsInAdditiveAnimationBlacklist(NSString *keyPath) {
```

Comment:
IMO something like `isAnimationKeyAlwaysNonadditive` would be a better name.

Reply:
Done.